docker/filebeat/filebeat.yml

```
@@ -25,190 +25,142 @@ filebeat.autodiscover:
       templates:
         - condition:
             contains:
-              docker.container.image: "apm-server"
+              docker.container.name: "opbeans-"
           config:
             - type: container
               paths:
-                - '/var/lib/docker/containers/*/*.log'
-              containers.ids:
-                - "${data.docker.container.id}"
+                - "/var/lib/docker/containers/*/${data.docker.container.id}-json.log"
+              include_lines: ['^{']
+              tail_files: true
+              processors:
+              - add_tags:
+                  tags: [json]
+                  target: "parser_type"
+              - decode_json_fields:
+                  fields:
+                    - message
+                  target: "json"
+                  overwrite_keys: true
+                  add_error_key: true
               fields_under_root: true
-              json.keys_under_root: true
-              json.overwrite_keys: true
-              json.add_error_key: true
-              json.message_key: message
         - condition:
             contains:
-              docker.container.image: "filebeat"
+              docker.container.name: "opbeans-"
           config:
             - type: container
               paths:
-                - '/var/lib/docker/containers/*/*.log'
-              containers.ids:
-                - "${data.docker.container.id}"
-              fields_under_root: true
-              json.keys_under_root: true
-              json.overwrite_keys: true
-              json.add_error_key: true
-              json.message_key: message
+                - "/var/lib/docker/containers/*/${data.docker.container.id}-json.log"
+              tail_files: true
+              multiline.pattern: '^[[:blank:]]'
+              multiline.negate: false
+              multiline.match: after
+              exclude_lines: ['^{']
+              processors:
+              - add_tags:
+                  tags: [no_json]
+                  target: "parser_type"
         - condition:
             contains:
-              docker.container.image: "heartbeat"
+                docker.container.name: "kibana"
```

Not sure whether this is a cosmetic ident or it might imply something else
```suggestion
              docker.container.name: "kibana"
```